### DISCUSSION TITLE
Mythic Dungeon compromise:  Replace the objectively better loot with a different reward system.

### DISCUSSION THREAD

**Discussion by Vyperhand** [chromiecraft/chromiecraft#5865] [Suggestions]
Because this is a hot topic - suggesting a possible nicely peacemaking alternative.

Rather than have the currency spendable on objectively better gear, the following alternatives could be offered as rewards:

Any heroic gear - allow people to pick pieces that have not dropped for them normally.
Gems
Enchant Scrolls
Mounts
Pets

Plus as many more ideas as you can come up with that are anything that is NOT just objectively better gear.

==
ETA -- Thinking about it, this would be a good place to put in some of the chase items that are incredibly hard to get via RNG sometimes - things like Baron Rivendare's Deathcharger and such.  I can imagine people would be happy to earn 20 of this currency (4 weeks worth)  - or more - and trade those four weeks for a guaranteed chase item.

Maybe even some super exclusive vanity rewards that aren't even available through the donor shop - sky's the limit there - cross faction mounts as an example maybe?

**Comment by heyitsbench** [comment]
I would be careful with the 'any heroic gear' part. I'm not all too certain what items look like for TBC, but I know in Classic's catch-up system (titan runes alpha/beta) even some items from a couple phases back are not acquirable through the catch-up mechanics (illustration of the dragon soul for instance). I feel if there are other hard hitting items like that in loot tables for TBC, they should not be acquirable through the mythic score system. But again, not sure if there are any items like that or what they might be.

**Reply by Vyperhand** [reply]
I am not a gear expert, so yeah - painting in broad strokes to get the idea out there to be refined and iterated upon.   :) 

Edit 2:  Moving the details to the original post so they're not lost in the exploding thread.

**Comment by Zelfie** [comment]
Gather more feedback before making insane changes like this.
Add Quags eye to loot dragon.
Remove bonus stat nonsense.

**Reply by heyitsbench** [reply]
> Gather more feedback before making insane changes like this.

The changes are not live yet. Consider this the feedback gathering stage.

**Reply by Vyperhand** [reply]
Please don't pollute a reasonable request with insults.  That makes it much less likely to be adopted.

**Comment by Never6** [comment]
Imo mythic tuned content is fine.
Seems a little unwarranted because why not just make heroics as hard as you want mythics to be
The issue is rewards
I think a far better option is, instead of 1 enchant per item it's a gem that players can choose
7sp isn't much on one item, sure it's a lot if you replace every piece but that isn't guaranteed. Not to mention some classes don't even use all epic pieces from heroics if any even do.
People being able to pick gem is better for a couple reasons

Players can choose what they want.
It's unfair for say a warrior to get 14ap when 8str or 13 crit rating gems are better.
8str is prob 20ap in full buffs
8dodge rating is also bad, from a bear main perspective. Dodge rating is inferior to agi per point not to mention you get more than just % dodge from agility.
Even if bears go full avoidance and get their Stam from gear not gems agility is better without question. And it doesn't take a bear main of 10 years to tell you that.
Having +1 gem means blue gear get buffed which most classes use at least some dungeon pieces ( at least the classes I play do) and since you can pick gem you might be able to change gear
Say a plate tank replaces 2 pieces of heroic gear with mythic gear and they get +2 gem slots, they use defense rating gems. This is what? +20defense for free? Now they might be 20points over cap which can let them potentially change gear slots for more threat or more stam
Off topic but
This content seems like something players should know about sooner than 3 days before implementation
Cause say majority hate it, what do you do?
Scrap the idea? You just wasted your time then.
If players knew about it sooner, you could get a feeling for how players feel about it and figure out if it's worth investi ng time into.
Cause now you box yourself into a corner. It's too late to change anything no amount of critic or backlash is going to change anything

**Reply by heyitsbench** [reply]
> Three days before worldbosses are added, not the mythic modes.

Three days before worldbosses are added, not the mythic modes.

**Reply by Never6** [reply]
Ok thought they came together
Still think players should know about them sooner
What happend to polls?

**Reply by heyitsbench** [reply]
> What happend to polls?

As I said on Discord, "Because apparently the people who vote for the unpopular options got upset that democracy was a thing"

**Reply by Never6** [reply]
> > What happend to polls?
> 
> As I said on Discord, "Because apparently the people who vote for the unpopular options got upset that democracy was a thing"

Not in the discord

If people are mad that their unpopular take didn't get implemented then thats on them
And it's better for players to have some input than none at all

**Reply by Never6** [reply]
> > Three days before worldbosses are added, not the mythic modes.
> 
> Three days before worldbosses are added, not the mythic modes.

Worldbosses?
Doomwalker is t5 
His caster wep > gruul sword 
And nathrezim mindblade too

**Reply by Hihilist** [reply]
They are almost equal, so I would say it is just 3rd option. At least it does not invalidate t4 weapons as it could be though.
![obraz](https://github.com/chromiecraft/chromiecraft/assets/71841180/5b112ccb-9bc1-4e3f-a0cb-c7d50d71a83c)

**Reply by Never6** [reply]
> They are almost equal, so I would say it is just 3rd option. At least it does not invalidate t4 weapons as it could be though.
> ![obraz](https://github.com/chromiecraft/chromiecraft/assets/71841180/5b112ccb-9bc1-4e3f-a0cb-c7d50d71a83c)

Yes they are marginally close
But compare it to a heroic weapon
You also have to consider that 2 of those weps aren't even out when world bosses release.
You also have to consider that world bosses on cc are NOT blizzlike
 3day lockout means people can get those items much easier. Items that are supposed to be catchup for players who didn't do t4 which is why they are boe without changes. If you really want a t5 world boss in t4 then release him with Kara 
World bosses should have a 7day lockout without question
Imo it does invalidate those weapons because it's better than those two. Why would you use gruul sword as a lock when the dagger is better. Prob same for mage
Some casters can't even use swords, so the wait to get mindblade as a boomie isn't even worth it cause you have 3months to farm doomwalker. If you don't get it then the world hates you
So staff wanted to follow classic release
Doomwalker released in phase 4 btw

**Comment by Cooties-cc** [comment]
I personally had no qualms over mythic option on raids or dungeons, however when it was made public that the rewards from them also came with custom stats, that felt like a bait and switch from the original purpose. The bonus points always were described in a way that they were there to fill gaps in gear for items that just didn't drop, or you never won. Now it just seems like a odd way to slightly mimic hardmodes of raids in later content (or similar to token based upgrades from vendors). 
It now makes any piece of pre-raid gear objectively worse than the ones that comes with bonus stats. Those who already spent a ton of time farming said items, now feel (perhaps personal) slapped in the face with better upgrades without being from an actually higher level of content phase. Now if people want to push for the best gear in prep for raids, they are forced to re-run all the content they had been farming for a small stat upgrade. It's also to a degree of being so minimal, yet mathematically higher, that getting said upgrades doesn't even feel good. 
There is also another problem with the new items, which is having to re-gem and re-enchant all those items, which is a financial burden. Of course some have been playing for so long that gold isn't really an issue, and some may claim that those who do grind for this are already set on spending that much for a gold sink. I just didn't find it enjoyable having to not only vendor my fully kitted gear, but have to pay for the new one too. 

So where's the positive to this? Well for starters, I would suggest just going back to the normal items without the bonus stats. It's still a nice thing to get a gap filled every 5 days, which otherwise might take months of no luck (in rolls or drops). It also isn't an issue since the item being bought isn't being a replacement to the same item; it's an authentic upgrade. I don't think many people were opposed to the idea of spending mythics points on gear from heroics, at least until the stat change. 
I also suggested a couple of ideas (I don't think have been mentioned yet?) that could be added to the vendor for people to buy. One of which was adding the lvl 70 perm enchant items that were from wotlk crafting. 
![image](https://github.com/chromiecraft/chromiecraft/assets/62584589/e116b23c-5bcd-488c-8c6a-51281b6382bb)
These items are not custom, but they also aren't available by normal means. They aren't a massive upgrade (armor kits) to what already exist from crafters, and it's not to a degree that would shut out those crafters. People would often still buy the cheap armors over spending tons of gold for the upgraded ones, or more so running 5 mythics. There are also other items that add stats, but again, not overtly overpowered. These items also last past the life of mythics, since once raids come out, mythics are technically obsolete. If adding a very high lvl enchant to some slots is a thing, mythics are still going to be useful to run (unless added to mythic raid vendor table). 
The second one which I think has maybe been brought up, but not for this, was adding in BoA rep tokens. 
![image](https://github.com/chromiecraft/chromiecraft/assets/62584589/0be90ae3-2aeb-4cd8-a1ad-8efe31ff5096)
These were added for wotlk reps from the badge vendors, and used for alts that wanted to speed up the rep process for rewards (or for us, keys to heroics). If there was a way to add these, but for TBC reps, I think they would be a great addition that also keeps mythics alive past raids. 
Ofc other things like vanity rewards are also 100% fine by my books. Just nothing that can be obtained by working ingame for such items (no free al'ars, etc). 

In closing, if none of these additions were of value, I would say the bare minimum would be to remove the bonus stats. I think custom gear is somewhat pushing it towards a funserver, and while it was stated a while back that it's not 100% blizzlike server, I would still enjoy it to be in some regards. Gear being one. I don't mind the enhanced difficulty, or the custom fights, but it is bordering into something along more TBC+ than wotlk tbc. It's not quite the same as the custom bosses for fun that were added, and if in the future further custom content is planned beyond this, it might start turning people off when it affects the main pve/pvp aspects of the game.

**Reply by balleny** [reply]
I dislike the idea of adding "improved" gear.
I like the idea of adding e.g. the belt bucket **BUT** if you use it on an piece of gear, you shouldnt be able to use it in arena during TBC, and each belt bucket bought should double the token costs of the next one (to prevent an inflation of the belt buckets, to make people think on which item they wanna use it, etc.)

another thing, depending on tuning, which I could imagine is:
adding a one-time buy per char and through that allow e.g. 36 tokens to be exchanged for a 36 slot bag ONCE per char - and maybe if we do keep this mythic system for raids later on, repeat it in a similar manner once per tier (which would mean one bag for hc and one bag for t4, t5 and t6 respectively

**Comment by quigiboo** [comment]
The mythics are a fun idea with additional mechanics, I certainly prefer that to simply increasing the numbers. We tested the 3 that are available on the PTR last night, and although they require some tweaking, I think it could be enjoyable 5 man content later in the expansion.
However I would much prefer this system without custom loot rewards, I would be fine with just being able to purchase some items as they are, kind of bad luck protection, or possibly the reward points could be account bound? 
I would also like to add I appreciate the amount of work that has to go into adding custom mechanics, I am grateful for the efforts.

**Comment by BaoZaiz** [comment]
Kind of agree with Cooties, been spent so much time and effort to get all so call "pre-raid" bis gears ...

But with Mythic loot, all turn meaningless now ....

**Reply by Aethereus-ManaOnion** [reply]
100% agree

**Comment by Hihilist** [comment]
After the announcement and some follow-up info on the Mythic loot, can still say that all heroic-specific gear should be added to Chromie NPC as a possible reward for Mythic difficulty.
Reason? RNG, bugged loot pools, and also the fact that some rare gear is more sought after than epic gear.

Adding vanity stuff that can be obtained in the game already but from Mythic difficult is a hard no.
_Reason?_ Middle finger to those that spend their time in-game to get those items + hard to balance with prices (e.g. Rivendare mount suggested can be farmed endless amount of time per day with or without breaks between dung lockout). If someone is a completionist, they should farm stuff already how it was enabled or wait for wrath content that will trivialize the farm.

Adding small stat bonuses to gear from Mythic difficulty is okay, they are not big upgrades so if someone wants to min-max, they have a way to do it. Just the fact that Mythic difficulty will not only be scaled but also with added new mechanics seems weird (it was not supposed to be that but like I am saying below everyone has Heroic on the farm, so maybe just tuning was not enough).
_Reason?_ The previously announced system allowing players to nail down RNG to get a missing piece of gear was good IF we got Heroic AND Mythic difficulties at the same time AND Heroic was not buffed like it is now. Adding the old system to the current environment makes no sense since most people already have almost everything they needed or have friends to help to farm. So from the development point of view that you create something for all to be interested, CC staff had to tweak the idea to add some incentive (please, stop bullshit like "bragging right" or "feeling of satisfaction", it is the future where put effort = reward obtained).

Must agree with Never that while stat bonuses are cool, some of them are bad from players' point of view and the class they play. Adding the option to pick a bonus or add an extra prismatic socket (there are some GM spells that can do that) that can hold any existing gem seems better.
_Reason?_ As he said, some classes benefit more from pure agility, strength, stamina, intellect, and critical rating, so narrowing options to SP/AP does sound not very convincing to aim for gear, either if you are min-maxing or if you want something good for your struggle.
Dodge rating is the least tanky stat that could be picked so need to be replaced for sure.

Cannot agree with Cooties (except one thing). People got their pre-raid bis so they do not need to care about new loot since their current gear is enough to clear the raid, no? The only force that pushes them toward the new pieces is their personal urge to either min-max or make content easier on Mythic/trivial on Normal/Heroic (afaik there was guild testing Mythic raids in normal gear and they had no problems with that, so new Mythic gear is not mandatory to beat Mythic raids). Stat bonuses should not be significant as Mythic is a custom addition, optional, only for those who want it. Otherwise, we create a hard FOMO for most of the server that does not need this feature.
The loop that makes people enchant again new gear is also good cause it keeps the server's economy alive (farmers farm to earn gold, crafters craft to earn gold).
Adding Wrath crafting upgrades is a hard pass - it is not a future-proof concept as the Mythic difficulty would have to be revisited again when we hit Wrath. Otherwise, crafters will lose their perks and gold-making possibilities. And it is not enough to cover all the gear for all the classes. Plus again, stats should not be significant, so the majority of players should not feel a must to participate in Mythic difficulty addition.
BoA rep tokens for TBC factions sound good - helps with maintaining the alts, keeping Heroic/Mythic difficulty more alive.

**Reply by Cooties-cc** [reply]
I'll just chirp in on the part that addressed me. 

I agree that the bonus stated gear is optional and not a requirement to do normal raids. I mentioned it only as something that "if people want to push for the best gear; the stats are minimal yet mathematically higher" but I never claimed it was do or die. It isn't enough to force people to run mythics to get stats that will allow to clear normal, or even possibly mythic raids. The normal gear should be more than enough judging from previous vanilla raid tuning (and even ubrs nightmare tuning). 

The FOMO also addressed another thing I hadn't included in my post (it was midnight and I was burntout). The new enchanted mythic dungeon gear, or mythic raid, has the possibility to create a social stigma against players. It's very clear to see if someone has the mythic version or not, and it could cause ppl to treat those who didn't min-max the small stats differently. Of course this won't really happen in guilds, more just pugs n what not. However I could see some ppl mock others for not having the alt version. 

I do understand that some of the craft items hurt tailors and LW the most, since armor kits are of their biggest sales. I could agree that those could be side stepped or alt items perhaps included. However the belt buckle/BS ones I don't see being that harmful. The belt buckle being the one in the list not even possible atm. I had also suggested in guild comms that perhaps other consumed materials could be included possibly, given the idea it wasn't hurting the econ. Something that had people want to revisit content to get materials/consumes, where as gear has hard limitations. Since we don't get reward bags at the end of the RDF at 70, maybe they could add in features for lower cost crafting materials or such (as they did before with rdf bags) to buy from vendor. The future thing isn't an issue either, since we had TBC crafting materials on Robo in gad during vanilla, since some enchants and stuff needed them. It was a non-issue to have TBC mats/items before TBC since they were removed on TBC launch, and the vendor taken away (in this case just remove from vendor as option). The same thing can be applied when wotlk content launches, as a way to prevent stockpiling. 

Also to slightly address the comment _"Plus again, stats should not be significant, so the majority of players should not feel a must to participate in Mythic difficulty addition."_ I agree that mythics shouldn't be a forced stepping stone into normal raids. It should be treated as opt content for those wanting more out of their time spent. In regards prev to the wotlk armor kits, it isn't as big as a jump as you may think, and wouldn't really feel "forced" to get them. 
![armor](https://github.com/chromiecraft/chromiecraft/assets/62584589/f850a6bf-ad33-41e6-9247-14895399b9e2)
The difference in stats is very minimal, and wouldn't really pigeonhole people into feeling forced to get them. However I do agree that adding in items that only hurt a portion of crafters isn't a great feel either, since people would often just stockpile mythic points to only buy these enchants for future content. I know as a JC if they added gems that were just 1pt above my cuts, I'd be upset. I just liked the controlled enchant aspect since it can be applied to any piece of gear, that didn't harm the already existing piece of gear, and can scale with any content going forward.

**Comment by InvictaChromie** [comment]
> Because this is a hot topic - suggesting a possible nicely peacemaking alternative.
> 
> Rather than have the currency spendable on objectively better gear, the following alternatives could be offered as rewards:
> 
> Any heroic gear - allow people to pick pieces that have not dropped for them normally. Gems Enchant Scrolls Mounts Pets
> 
> Plus as many more ideas as you can come up with that are anything that is NOT just objectively better gear.



This sounds good to me !

**Comment by Aethereus-ManaOnion** [comment]
Adding boosted loot to such a mechanic is a gigantic middle finger to those who care about being prepared for raids as pre-bis. First of all you have to farm the same F****G content AGAIN. Who the **** wants to run those god-forsaken dungeons on yet another difficulty level after having ran them endlessly while levelling, initial gearing, rep grinding to get heroic keys, and then heroics. At least you dont need attunement for those mythics...

Second of all, if you introduce a mechanic that gives better loot, then it stops being optional. It becomes necessary, because everybody and their mother knows that raids will be optimized around the top-end-players to provide a challenge for them. So if you give hardcore sweaty players access to like 100 more ap each with all those items combined, then it will have to be reflected in the tuning of all raids as long as this gear is not obsolete.

HARD PASS, PLEASE RECONSIDER

TL;DR: make it optional content, not an ugly necessity

**Reply by Never6** [reply]
I think they are making raids have selectable hardmode. 
So you can do either normal Kara or you can toggle hardmode Kara
I think that is a bad idea but that is a different discussion. 
So mythic mode is optional to a degree because you can just do normal modes and not hardmodes/custom difficulty.

**Comment by Amrx2** [comment]
https://forms.gle/E8r56YFBzJ1GzTmA9

For those of you not aware / seen it in game / saw it on the Discord.
